cookbooks/aws-parallelcluster-byos/recipes/install_user.rb

```diff
@@ -34,6 +34,21 @@
   shell '/bin/bash'
 end
 
+# Reserve byos system users that will be replaced at node configuration time.
+(0..9).each do |i|
```

Comment:
Minor: do we want to create user from 410-420 or 410-419? From design doc, the range may be (0..10)

Reply:
410-420 is desired.